packages/core/stories/patterns/search/search.stories.tsx

```diff
@@ -0,0 +1,320 @@
+import {
+  Input,
+  Button,
+  FormField,
+  FormFieldLabel,
+  FormFieldHelperText,
+} from "@salt-ds/core";
+import { CloseIcon, EditIcon, SearchIcon } from "@salt-ds/icons";
+import { Meta, StoryFn } from "@storybook/react";
+import { ChangeEventHandler, useRef, useState } from "react";
+
+export default {
+  title: "Patterns/Search",
+} as Meta;
+
+export const DefaultIcon: StoryFn<typeof Input> = (args) => {
+  const [value, setValue] = useState("");
+  const inputRef = useRef<HTMLInputElement>(null);
+
+  const focusOnInput = () => {
+    if (inputRef.current) {
```

Comment:
This can be simplified, `inputRef` should be used instead of the `ref` prop and then you can use the ref to set the focus rather than querying the DOM on a fragile class name